packages/core/test/state.test.ts

```diff
@@ -331,14 +331,14 @@ describe('State', () => {
     });
   });
 
-  describe('State.create()', () => {
+  describe('machine.createState()', () => {
```

Comment:
It looks like with this change the `State.create` method should be removed and all its occurrences should be replaced with `machine.createState`